content/get-started/getting-started-with-git/configuring-git-to-handle-line-endings.md

```diff
@@ -107,29 +107,40 @@ You'll notice that files are matched—`*.c`, `*.sln`, `*.png`—, separated by
 
 When you set the `core.autocrlf` option or commit a *.gitattributes* file, you may find that Git reports changes to files that you have not modified. Git has changed line endings to match your new configuration.
 
-To ensure that all the line endings in your repository match your new configuration, backup your files with Git, delete all files in your repository (except the `.git` directory), then restore the files all at once.
-
-1. Save your current files in Git, so that none of your work is lost.
-  ```shell
-  $ git add . -u
-  $ git commit -m "Saving files before refreshing line endings"
-  ```
-2. Add all your changed files back and normalize the line endings.
-  ```shell
-  $ git add --renormalize .
-  ```
-3. Show the rewritten, normalized files.
-  ```shell
-  $ git status
-  ```
-4. Commit the changes to your repository.
-  ```shell
-  $ git commit -m "Normalize all the line endings"
-  ```
+To ensure that all the line endings in your repository match your new configuration, backup your files with Git, then remove/restore the files all at once with renormalized line endings.
+
+1. Save your current files in Git, so that none of your work is lost.  
+   ```shell
+   $ git add . -u
+   $ git commit -m "Saving files before refreshing line endings"
```

Comment:
Cleaning up some of the existing content here: let's lead with the "why" for these steps. Also, because these command blocks contain no output, I've removed the prompts per our [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#code-blocks). I've also added a directive to make this a little easier for readers to copy.

```suggestion
1. To ensure that none of your work is lost, commit any changes by running the following commands.

   ```shell{:copy}
   git add . -u
   git commit -m "Saving files before refreshing line endings"
```